plugins/UsersManager/vue/src/PagedUsersList/PagedUsersList.vue

```diff
@@ -276,7 +276,9 @@
                     showAccessChangeConfirm();"
                   :disabled="user.role === 'superuser'"
                   uicontrol="select"
-                  :options="user.login !== 'anonymous' ? accessLevels : anonymousAccessLevels"
+                  :options="
+                    user.login === 'anonymous' ? anonymousAccessLevels :
+                    (user.role === 'noaccess' ? onlyRoleAccessLevels : accessLevels)"
```

Comment:
The paged user list, currently shows all roles and capabilities in the role selection. Selecting (adding) a capability only works if the role is at least `view`, so this change will hide the capabilities in the selection unless the user has at least view permission.